lib/lotus/helpers/numbers/number_formatter.rb

```diff
@@ -0,0 +1,72 @@
+module Lotus
+  module Helpers
+    module Numbers
+      # Numbers
+      #
+      # @since x.x.x
+      class NumberFormatter
```

Comment:
@AlfonsoUceda Thanks for your comments. I didn't comment this class because I thought of it as a 'private' class that is only used internally. Therefore, it shouldn't appear in the public documentation.